psqlextra/compiler.py

```diff
@@ -6,13 +10,59 @@
 from django.core.exceptions import SuspiciousOperation
 from django.db.models import Expression, Model, Q
 from django.db.models.fields.related import RelatedField
-from django.db.models.sql.compiler import SQLInsertCompiler, SQLUpdateCompiler
+from django.db.models.sql.compiler import (
+    SQLAggregateCompiler,
+    SQLCompiler,
+    SQLDeleteCompiler,
+    SQLInsertCompiler,
+    SQLUpdateCompiler,
+)
 from django.db.utils import ProgrammingError
 
 from .expressions import HStoreValue
 from .types import ConflictAction
 
 
+def append_caller_to_sql(sql):
```

Comment:
**question**: Can we put this behind a Django setting flag, disabled by default? Not all users might want this feature enabled by default.